src/main/java/refinedstorage/apiimpl/util/Comparer.java

```diff
@@ -18,7 +18,9 @@ public boolean isEqual(ItemStack left, ItemStack right, int flags) {
         }
 
         if ((flags & COMPARE_OREDICT) == COMPARE_OREDICT) {
-            return isEqualOredict(left, right);
+            if (isEqualOredict(left, right)) {
```

Comment:
If items aren't registered in the ore dict that will return false even if both stacks are the same Item, Damage and NBT